app/views/users/cards/_submission.html.erb

```diff
@@ -5,10 +5,10 @@
   </td>
   <td>
     <h4 class='text-<%= submission.class.to_s.downcase %>'>
-      <i class='fa <%= submission.icon %>'></i> 
-      <%= link_to submission.title, after_update_object_path_for(submission) %>
+      <i class='fa <%= submission.icon %>'></i>
+      <%= link_to submission.title.html_safe, after_update_object_path_for(submission) %>
```

Comment:
Don't think we'd need `.html_safe` on title since we don't allow for RTF on title fields? Even with the experience one you stripped tags, so we should be fine here. Generally `.html_safe` is not actually that safe as if it's `nil`, it will cause the page to error out, so we should use it sparingly and only when we're certain that input will be present.